typify/tests/schemas/nested-newtype.json

```diff
@@ -0,0 +1,14 @@
+{
+  "title": "NestedNewType",
+  "type": "object",
+  "properties": {
+    "field": {
+      "type": "string",
+      "enum": [
+        "a",
+        "b"
+      ],
+      "maxLength": 25
```

Comment:
in this particular case, I'd rather that we do the `maxLength` validation on the enumerated values and just emit an `enum` for `NestedNewTypeField`. In other words: the `maxLength` is pointless so let's not bother with it. Or if it's relevant, let's either skip the generation of the enumerated values that are too long.